hledger-lib/Hledger/Data/Posting.hs

```diff
@@ -171,7 +171,7 @@ postingStatus Posting{pstatus=s, ptransaction=mt}
 -- | Implicit tags for this transaction.
 transactionImplicitTags :: Transaction -> [Tag]
 transactionImplicitTags t = filter (not . T.null . snd) [("code", tcode t)
-                                                        ,("desc", tdescription t)
+                                                        ,("description", tdescription t)
```

Comment:
The name "desc" were used for consistency with `Query.hs` where you use `desc:SomePayeeRegex`.